docs/tutorials/get-started-with-ngo.md

```diff
@@ -116,7 +136,7 @@ The **Enable Scene Management** for the `NetworkManager` setting allows the serv
 
 ## Add RPCs
 
-This section guides you through adding basic RPCs to the project. Save your scripts in your `Assets/Scripts/` folder.
+This section guides you through adding basic RPCs to the project. Save your scripts in your `Assets/Scripts/` folder. RPCs are used to call functions on remote clients or the server.
```

Comment:
The HelloWorldManager should be above this. 
Going to make some minor additional changes.